solutions/src/main/java/ru/sberbank/school/task07/ReverseOrderImpl.java

```diff
@@ -0,0 +1,24 @@
+package ru.sberbank.school.task07;
+
+import java.io.FileNotFoundException;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReverseOrderImpl implements ReverseOrder {
+
+    private final FileParser fileParser;
+
+    public ReverseOrderImpl(FileParser fileParser) {
+        this.fileParser = fileParser;
+    }
+    @Override
```

Comment:
&#39;METHOD_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1191573947>*